### PR TITLE
change trigger logic

### PR DESCRIPTION
**Pull request by naterg**
Change trigger logic to fire after one or more failures in the downtrigger time, taking into account the current polling interval

### CODE REVIEW

File: webseer_process.php [hunk 1]
```diff
@@ -185,7 +185,7 @@
 		if ($results['result'] == 0) {
 			$url['failures'] = $url['failures'] + 1;
 //			//////////if ($url['failures'] > $url['downtrigger'] && $url['triggered'] == 0) {
-			if ($url['failures'] >= $tf && $url['triggered'] == 0) {
+			if ($url['failures'] >= ($url['downtrigger'] * 60)/$poller_interval && $url['triggered'] == 0) {
```
Comment (netniV) [inline webseer_process.php:188]: Not sure that this right.  If you consider that downtrigger is 2 (down for 2 polls), then 2*60 = 120 / 300 (5 minute poller) seems wrong.

Comment (netniV) [reply webseer_process.php:188]: @cigamit may be able to (dis)agree with me on that.

Comment (naterg) [reply webseer_process.php:188]: Perhaps I'm misinterpreting the downtrigger value? The description in the web interface says "How many minutes the URL must be down", not how many polling cycles to be down. In that case if downtrigger is 2 minutes in a 5 minute polling cycle you should only need one failure to trigger (which is >= 2*60/300)...

Comment (netniV) [reply webseer_process.php:188]: OK, I'll roll with that.

**Review by netniV — CHANGES_REQUESTED** [review]
Please update the README.md with the original issue number and title.

**Comment by naterg** [comment]
updated README

**Review by netniV — APPROVED** [review]

**Comment by naterg** [comment]
thanks @netniV !

**Pull request merged**